src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalSummary.scala

```diff
@@ -25,15 +25,17 @@ object AnnualUpdateMinimalSummary {
   // TreeLossSummary form Raster[TreeLossTile] -- cell types may not be the same
 
   implicit val mdhCellRegisterForTreeLossRaster1
-  : CellVisitor[Raster[AnnualUpdateMinimalTile], AnnualUpdateMinimalSummary] =
-    new CellVisitor[Raster[AnnualUpdateMinimalTile], AnnualUpdateMinimalSummary] {
+    : GridVisitor[Raster[AnnualUpdateMinimalTile], AnnualUpdateMinimalSummary] =
+        new GridVisitor[Raster[AnnualUpdateMinimalTile], AnnualUpdateMinimalSummary] {
+      private var acc: AnnualUpdateMinimalSummary = new AnnualUpdateMinimalSummary()
 
-      def register(
+      def result: AnnualUpdateMinimalSummary = acc
+
+      def visit(
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)